packages/bpk-component-breakpoint/package.json

```diff
@@ -15,7 +15,7 @@
   "dependencies": {
     "bpk-mixins": "^17.10.4",
     "prop-types": "^15.5.8",
-    "react-responsive": "^3.0.0"
+    "react-responsive": "^4.1.0"
```

Comment:
Couldn't figure out what the actual breaking change here is from their changelog

Reply:
Link to changelog -> https://github.com/contra/react-responsive/blob/master/CHANGELOG.md#v400-20171116-0014-0000